<launch>
    <node pkg="can_node" type="odrive.py" name="odrive_controller" output="screen">
        <param name="interface" value="can0" />
        <param name="baudrate" value="250000" />
        <param name="odrive_dbc" value="$(find can_node)/odrive.dbc" />
    </node>
</launch>